specifyweb/frontend/js_src/lib/components/Syncer/syncers.ts

```diff
@@ -238,12 +238,23 @@ export const syncers = {
         const currentChildren =
           children[tagName] ?? children[tagName.toLowerCase()] ?? [];
         const child = currentChildren[0];
+
+        let hasCondition = false;
+        currentChildren.forEach((child) => {
+          if (
+            child.attributes.condition !== undefined &&
+            child.attributes.condition.length > 0
+          ) {
+            hasCondition = true;
+          }
+        });
```

Comment:
this code assumes specific structure from the input code
(it assumes that the child has a condition attribute and assigns some semantic meaning to it)
this function is used in many places - you can't just change it like this - it will affect all the other places where it's used

Instead, if you need to support multiple children, then modify the rule for the form definition to call syncer.xmlChildren instead of syncer.xmlChild